crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1334,6 +1476,7 @@ impl SymbolicationActor {
         let sources = request.sources.clone();
         let scope = request.scope.clone();
         let signal = request.signal;
+        let origin = request.origin.to_string();
```

Comment:
If you make `StacktraceOrigin` `Copy` then you don't need to create this variable here, but instead can pass `request.origin` straight into `record_symbolication_metrics()` if you update it to take the `StacktraceOrigin` instead of `String`.  I think that would be a nice improvement.
(strictly speaking you don't even need it to be copy and can pass a reference, but you might as well make it copy)

Reply:
The problem is that I can’t `move` `request` into the closure, however I now pass it as a typed enum instead of as a `String`, so thats at least a small improvement.

Reply:
huh, oh yes.  that requires moving entire request into the async closure, i overlooked that 😞    we're getting partial moves soon i think, though not sure that'd help here.